source/driver.cpp

```diff
@@ -62,7 +62,7 @@ void Driver::reading(void)
 
 #ifdef __MPI
     // (4)  divide the GlobalV::NPROC processors into GlobalV::KPAR for k-points parallelization.
-    Parallel_Global::init_pools();
+    Parallel_Global::divide_pools();
```

Comment:
The call of init_pools() change to divide_pools(), which looks not related to the Issue or the title of PR?